.circleci/config.yml

```diff
@@ -0,0 +1,81 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/qpp-conversion-tool
+    docker:
+      - image: circleci/openjdk:8-jdk-browsers
+    steps:
+      - checkout
+      - run: cd converter; mvn dependency:resolve
+      - run: wget -O sonarscanner.zip https://sonarsource.bintray.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-3.0.1.733.zip
+      - run: unzip -d . sonarscanner.zip
+      - run: wget -O get-pip.py https://bootstrap.pypa.io/get-pip.py
+      - run: sudo python get-pip.py
+      - run: sudo pip install ansible
+      - run: cp ./sonar-scanner.properties sonar-scanner-3.0.1.733/conf/sonar-scanner.properties
+      - run: mvn clean verify
+      - run: ./tools/bin/sonarqube.sh
+      - run: ./tools/bin/docker_deploy.sh
+      - run:
+          name: Creating artifacts
+          command: |
+            ARTIFACT_DIR=/tmp/artifacts;
+            mkdir $ARTIFACT_DIR;
+            cp converter/target/converter.jar $ARTIFACT_DIR;
+            cp commandline/target/commandline.jar $ARTIFACT_DIR;
+            mkdir -p $ARTIFACT_DIR/coverage/converter;
+            cp -R converter/target/site/jacoco $ARTIFACT_DIR/coverage/converter;
+            mkdir -p $ARTIFACT_DIR/coverage/rest-api;
+            cp -R rest-api/target/site/jacoco $ARTIFACT_DIR/coverage/rest-api;
+            mkdir -p ARTIFACT_DIR/coverage/commandline;
+            cp -R rest-api/target/site/jacoco $ARTIFACT_DIR/coverage/commandline;
+
+      - store_artifacts:
+          path: /tmp/artifacts
+
+  ui_dependencies:
+    working_directory: ~/qpp-conversion-tool/frontend
+    docker:
+      - image: circleci/node:7.10.0
+    steps:
+      - checkout
+      - run: curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+      - run: sudo dpkg -i google-chrome.deb
+      - run: sudo apt-get install jq
+      - run: sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
+      - run: rm google-chrome.deb
+      - run: cd frontend; rm -f yarn.lock && yarn
+      - save_cache:
+          key: dependency-cache
+          paths:
+            - ~/.cache/yarn
+
+  ui_deployment:
+      working_directory: ~/qpp-conversion-tool/frontend
+      docker:
+        - image: circleci/node:7.10.0
+      steps:
+      - checkout
+      - run: npm install -g protractor;
+      - run: npm install; npm test;
+      - run: yarn run build;
+      - run: aws s3 sync frontend/dist s3://qpp-conversion-tool-ui --delete
```

Comment:
Let's make sure this goes to our new AWS account

Reply:
Peter briefed me on that